students/robert/Session 6/Trapezoidal_Rule.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python 
+
+#import math
+
+"""
+def line(x):
+    function = 2*x + 5
+    return function
+
+def line(x,A,w)
+    function = A*math.sin(w*x)
+    return function
+"""
+
+def quadratic(x, A=0, B=0, C=0):
+    return A * x**2 + B * x + C
+
+def trapz(quadratic,a,b,n,A,B,C):
```

Comment:
Can you update your trapz function so that it can accept any function as an argument? Any function with any number of arguments? 

Think about how you can use *args and **kwargs to make this more flexible.